### ISSUE TITLE
Document how to use image_id: vs image: for Droplet.new()

### ISSUE THREAD

**Issue by hh** [digitalocean/droplet_kit#30]
I'm not sure where the names come from for calls to Droplet.new(name:) when I want to specify my own images.

I ended up using the image_id: in the name: parameter to accomplish what I wanted, but it was confusing.

**Comment by bobbytables** [comment]
To clarify, `image_id` _did_ work right? You're just looking for better documentation?

**Comment by hh** [comment]
Yea it works.

**Comment by hh** [comment]
well, not via image_id: I provided the image id to the name: parameter

**Comment by hh** [comment]
It's the image: parameter, which get's fed from apps using the library (like test-kitchen). Knowing how name names map to image id's would be great.

**Comment by bobbytables** [comment]
Yeah, right now the best way to see slugs is to retrieve only distribution images from `/images` 

https://developers.digitalocean.com/documentation/v2/#list-all-distribution-images

**Comment by hh** [comment]
I'm guessing no slugs are created by default for snapshot / application images.

**Comment by bobbytables** [comment]
Application images should also have slugs.

**Comment by VinceG3** [comment]
How do I actually do this with DropletKit, see the image ids? What's the magic spell? I see the api documentation link, but that doesn't tell me how to actually do it with the ruby gem. I don't understand why this is so hard.

**Comment by VinceG3** [comment]
This is the magic spell for anyone else coming across this:

`client.images.all.each{|i| puts "#{i.id}: #{i.distribution} #{i.name}" }`
